helm-charts/common/asr/templates/configmap.yaml

```diff
@@ -20,3 +20,4 @@ data:
   {{- else }}
   no_proxy: {{ .Values.global.no_proxy | quote }}
   {{- end }}
+  LOGFLAG: {{ .Values.LOGFLAG | quote }}
```

Comment:
On second thought, based on the [current python code logic1](https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L37C14-L37C20) and [logic2](https://github.com/opea-project/GenAIComps/blob/main/comps/dataprep/redis/langchain/prepare_doc_redis.py#L45), whenever we set the LOGFLAG env as a non-empty string, it will be treated as  boolean `True`.

So we need to change the template accordingly.

Reply:
Update each values.yaml file to address the issue, please help take a look.